WordPressAuthenticatorTests/GoogleSignIn/Character+URLSafeTests.swift

```diff
@@ -0,0 +1,15 @@
+@testable import WordPressAuthenticator
+import Foundation
+import XCTest
+
+class Character_URLSafeTests: XCTestCase {
+
+    func testURLSafeCharacters() {
```

Comment:
nit: this took me a minute or two to grok – the combination of `XCTAssertFalse` and `.inverted` messed with my brain – basically we're saying "everything in `Character.urlSafeCharacters` should also be in `CharacterSet.urlQueryAllowed`", right?

If so, could we `XCTAssertTrue(CharacterSet.urlQueryAllowed.contains(character.unicodeScalars.first!))`? Or does that not properly capture what we're after? 

Also, can we `XCTUnwrap` `character.unicodeScalars.first!` to avoid the tests crashing?

Reply:
> Also, can we XCTUnwrap character.unicodeScalars.first! to avoid the tests crashing?

Done. See ca5d0a00.

I was at peace with force unwrapping there because, given the static nature of the inputs, `unicodeScalars` would "never™" have been empty. Still, I do appreciate the consistency of handling the possibility just in case.

Reply:
@jkmassel 

> basically we're saying "everything in `Character.urlSafeCharacters` should also be in `CharacterSet.urlQueryAllowed`", right?
>
> If so, could we `XCTAssertTrue(CharacterSet.urlQueryAllowed.contains(character.unicodeScalars.first!))`? Or does that not properly capture what we're after?

Yes, what I'm after here is to make sure that the two collections map 1:1.

I went for the `inverted` approach to cover the case were `CharacterSet.urlQueryAllowed` is a subset of `Character.urlSafeCharacters`. But with fresh eyes I now see that the straightforward approach you suggest has the same result. 😅 👍

Reply:
See a3af7d4f

Reply:
Actually, I just realized that the test can be made simpler and maybe clearer by using `SetAlgebra`.

See 811955fc